app/controllers/sessions_controller.rb

```diff
@@ -1,8 +1,13 @@
 class SessionsController < ApplicationController
+  cattr_accessor :code
   def create
+    SessionsController.token_code(params[:code])
     party = Party.find_by(identifier: request.env["omniauth.params"]["url"]) if request.env["omniauth.params"]["url"]
     if user = User.find_by(spotify_id: spotify_params(request.env['omniauth.auth'])[:spotify_id])
```

Comment:
Why not use User.find_or_create_by(spotify_params) here?

Reply:
Using find or create by now